sched/credit.cpp

```diff
@@ -524,21 +524,6 @@ int get_pfc(
         }
     }
 
-    // is result from old scheduler that didn't set r.app_version_id correctly?
-    // if so, use WU estimate (this is a transient condition)
-    //
-    if (r.app_version_id == 0 || r.app_version_id == 1) {
-        if (config.debug_credit) {
-            log_messages.printf(MSG_NORMAL,
-                "[credit] [RESULT#%lu] missing app_version_id (%ld): returning WU default %.2f\n",
-                r.id, r.app_version_id, wu_estimated_credit(wu, app)
-            );
-        }
-        mode = PFC_MODE_WU_EST;
-        pfc = wu_estimated_pfc(wu, app);
-        return 0;
-    }
-
```

Comment:
Do we still need to handle the case where r.app_version_id == 0?  Not all projects have updated to the new code yet so having this fall back would continue to support their transition gracefully.